### DISCUSSION TITLE
Practica css

### DISCUSSION THREAD

**Discussion by DaveOval** [Launch-X-Latam/MisionFrontEnd#324] [Show and tell]
Que onda camaradas les quiero enseñar como va la practica de css le he estado batallando mucho con las imagenes ya que quiero que el fondo ocupen la mitad de la pantalla pero solo ocupan una parte y luego quiero también implementar la disposicion para moviles, para el grid de ciertas partes utilice una herramienta que te genera el acomodo .

Aqui el repositorio: https://github.com/DaveOval/LaunchX_Vaccination

Aqui la pagina desplegada: https://daveoval.github.io/LaunchX_Vaccination/

**Comment by HoracioTorrez** [comment]
Te quedo super, una pregunta, ¿como haces para que el repositorio se convierta en una pagina desplegada? gracias

**Reply by DaveOval** [reply]
en tu repositorio

1. te vas a las opciones.
2. en la opción de github pages la seleccionas.
3. seleccionas la rama (normalmente es el master)
4. le das en guardar y listo (a veces tarda en cargar pero te da un link con tu pagina desplegada)

**Reply by HoracioTorrez** [reply]
muchas gracias...saludos!
